dpppt-backend-sdk/dpppt-backend-sdk-ws/src/main/java/org/dpppt/backend/sdk/ws/config/MultipleJWTConfig.java

```diff
@@ -148,7 +148,7 @@ protected void configure(HttpSecurity http) throws Exception {
           .cors()
           .and()
           .authorizeRequests()
-          .antMatchers(HttpMethod.POST, "/v1/exposed", "/v1/exposedlist", "/v1/gaen/exposed")
+          .antMatchers(HttpMethod.POST, "/v1/gaen/exposed", "/v2/gaen/exposed")
```

Comment:
Thinking about the future, why not match with a * the first part of the url (namely the version)?

Reply:
I like to have it explicit